flutter/lib/src/event_processor/replay_event_processor.dart

```diff
@@ -12,9 +12,15 @@ class ReplayEventProcessor implements EventProcessor {
 
   @override
   Future<SentryEvent?> apply(SentryEvent event, Hint hint) async {
-    final shouldCaptureReplay = event.eventId != SentryId.empty() &&
+    final isErrorEvent = event.eventId != SentryId.empty() && 
         event.exceptions?.isNotEmpty == true;
 
+    final isFeedbackEvent = event.eventId != SentryId.empty() &&
+        event.type == 'feedback';
+    final isWidgetFeedbackEvent = hint.get('isWidgetFeedback') == true;
+
+    final shouldCaptureReplay = isErrorEvent || (isFeedbackEvent && !isWidgetFeedbackEvent);
```

Comment:
wouldn't this also trigger when you send the feedback in the widget?

Reply:
ah yes, condition should be fine

- if the manual api was used without widget -> capture replay
- if the widget was opened -> capture replay
  - if then a feedback is sent -> dont capture replay